test/test_cases/TEST_ID_11_conjur_authn_failure.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euxo pipefail
+
+create_secret_access_role
+
+create_secret_access_role_binding
+
+export CONJUR_AUTHN_LOGIN_KEY_VALUE="CONJUR_AUTHN_LOGIN host/some-policy/non-existing-namespace/*/*"
```

Comment:
what does this extra space in `CONJUR_AUTHN_LOGIN host/some-policy/non-existing-namespace/*/*"` do? Does it set CONJUR_AUTHN_LOGIN: host/some-policy/non-existing-namespace/*/*, if so, how?

Reply:
explain face to face